<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FFTTestWgtClass</class>
 <widget class="QWidget" name="FFTTestWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1256</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FFTTestWgt</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QGroupBox" name="groupBox_baseSig_chart">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1000</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1980</width>
         <height>440</height>
        </size>
       </property>
       <property name="title">
        <string>基频信号绘制</string>
       </property>
       <layout class="QHBoxLayout" name="HLayout_baseSig_chart"/>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_baseSig_panel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>440</height>
        </size>
       </property>
       <property name="title">
        <string>基频信号</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_baseSig_Freq">
              <property name="text">
               <string>频率（HZ）</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dSBox_baseSig_Freq">
              <property name="minimum">
               <double>20.000000000000000</double>
              </property>
              <property name="maximum">
               <double>200000.000000000000000</double>
              </property>
              <property name="value">
               <double>969.299999999999955</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_baseSig_StartT">
              <property name="text">
               <string>开始时间（ms）</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dSBox_baseSig_StartT">
              <property name="maximum">
               <double>5000.000000000000000</double>
              </property>
              <property name="value">
               <double>200.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_baseSig_TimeLength">
              <property name="text">
               <string>信号时间长度（ms)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dSBox_baseSig_TimeLength">
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="value">
               <double>300.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_baseSig_Amp">
              <property name="text">
               <string>振幅(μPa)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dSBox_baseSig_Amp">
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>200000.000000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_baseSig_SampleRate">
              <property name="text">
               <string>采样频率（HZ）</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="dSBos_baseSig_SampleRate">
              <property name="maximum">
               <number>500000</number>
              </property>
              <property name="value">
               <number>44100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_drawBaseSignal">
          <property name="text">
           <string>绘制基础信号</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QGroupBox" name="groupBox_modSig_chart">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1000</width>
         <height>315</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1980</width>
         <height>440</height>
        </size>
       </property>
       <property name="title">
        <string>调制信号绘制</string>
       </property>
       <layout class="QHBoxLayout" name="HLayout_modSig_chart"/>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_modSig_panel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>315</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>440</height>
        </size>
       </property>
       <property name="title">
        <string>调制信号</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_modSig_Freq">
              <property name="text">
               <string>频率（HZ）</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dSBox_modSig_Freq">
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="value">
               <double>25.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_modSig_StartT">
              <property name="text">
               <string>开始时间（ms）</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dSBox_modSig_StartT">
              <property name="maximum">
               <double>5000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_modSig_TimeLength">
              <property name="text">
               <string>信号时间长度（ms)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dSBox_modSig_TimeLength">
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="value">
               <double>1000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label_modSig_Depth">
              <property name="text">
               <string>调制度（%）</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dSBox_modSig_Depth">
              <property name="maximum">
               <double>100.000000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_modSig_SampleRate">
              <property name="text">
               <string>采样频率（HZ）</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="dSBos_modSig_SampleRate">
              <property name="maximum">
               <number>500000</number>
              </property>
              <property name="value">
               <number>44100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_modSig_Type">
          <property name="title">
           <string>调制信号类型</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QRadioButton" name="radioButton_modSig_Type_sin">
             <property name="text">
              <string>正弦波</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_modSig_Type_square">
             <property name="text">
              <string>方波</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_modSig_DrawContent">
          <property name="title">
           <string>绘制内容</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QCheckBox" name="checkBox_modSig_drawModSig">
             <property name="text">
              <string>调制信号</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_modSig_drawSigAfterMod">
             <property name="text">
              <string>调制结果</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_drawmodSig">
          <property name="text">
           <string>绘制调制信号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_saveModResultToDCenter">
          <property name="text">
           <string>存储调制信号（D_Center）</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_20">
     <item>
      <widget class="QGroupBox" name="groupBox_FFT_chart">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1000</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1980</width>
         <height>440</height>
        </size>
       </property>
       <property name="title">
        <string>FFT结果</string>
       </property>
       <layout class="QHBoxLayout" name="HLayout_FFT_Chart"/>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_FFT_panel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>440</height>
        </size>
       </property>
       <property name="title">
        <string>FFT解析</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox_FFTFrom">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>52</height>
           </size>
          </property>
          <property name="title">
           <string>FFT来源</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QRadioButton" name="radioButton_FFTFrom_zhihu">
             <property name="text">
              <string>网友</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_FFTFrom_FFTW">
             <property name="text">
              <string>FFTW</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_FFTFrom_cudaFFt">
             <property name="text">
              <string>CudaFFT</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_21">
          <item>
           <widget class="QLabel" name="label_FFTStartTime">
            <property name="text">
             <string>输入开始时间(ms)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="dSBox_FFTstartTime">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>88</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>88</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QLabel" name="label_FFTinputPointInPowerOf2">
            <property name="text">
             <string>输入点数对应2的次数</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="dSBox_FFTInputPointInPowerOf2">
            <property name="minimumSize">
             <size>
              <width>88</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>88</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="value">
             <double>12.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QLabel" name="label_FFTSampleRate">
            <property name="text">
             <string>采样率(HZ)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="dSBox_FFTSampleRate">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>88</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>88</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>500000.000000000000000</double>
            </property>
            <property name="value">
             <double>44100.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_drawFFTResult">
          <property name="text">
           <string>绘制FFT结果</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_drawIFFT">
          <property name="text">
           <string>绘制IFFT结果</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_FFTInputPointCount">
          <property name="text">
           <string>采样点数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_FFTWindowLength">
          <property name="text">
           <string>窗口时长：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_FFTFreqDelta">
          <property name="text">
           <string>频率间隔：</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
